### PR TITLE
daemon: fix default array for CRUSH_LOCATION

### PR DESCRIPTION
**Pull request by leseb**
Fixes: https://github.com/ceph/ceph-docker/pull/732 and https://github.com/ceph/ceph-docker/issues/509
Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test ceph_ansible-luminous-centos7-dmcrypt_journal_collocation

**Comment by leseb** [comment]
jenkins test ceph_ansible-jewel-centos7-cluster

File: ceph-releases/jewel/ubuntu/14.04/daemon/osd_scenarios/osd_disk_activate.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ function osd_activate {
   else
     OSD_WEIGHT=$(df -P -k "$OSD_PATH" | tail -1 | awk '{ d= $2/1073741824 ; r = sprintf("%.2f", d); print r }')
   fi
-  ceph "${CLI_OPTS[@]}" --name=osd."${OSD_ID}" --keyring="$OSD_KEYRING" osd crush create-or-move -- "${OSD_ID}" "${OSD_WEIGHT}" "${CRUSH_LOCATION}"
+  ceph "${CLI_OPTS[@]}" --name=osd."${OSD_ID}" --keyring="$OSD_KEYRING" osd crush create-or-move -- "${OSD_ID}" "${OSD_WEIGHT}" "${CRUSH_LOCATION[@]}"
```
Comment (mikegrass) [inline ceph-releases/jewel/ubuntu/14.04/daemon/osd_scenarios/osd_disk_activate.sh]: osd_common.sh and osd_directory.sh should also be updated to expand the CRUSH_LOCATION array (by adding  `[@]`).

Comment (leseb) [inline ceph-releases/jewel/ubuntu/14.04/daemon/osd_scenarios/osd_disk_activate.sh]: @mikegrass oops, thanks, done in https://github.com/ceph/ceph-docker/pull/734

**Pull request merged**